projects/cli-files/go-cat/cmd/root.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+)
+
+func Execute() error {
+
+	numberLine := flag.Bool("n", false, "Add number in front of each line")
+	help := flag.Bool("h", false, "Show help")
+	helpFilePath := "assets/help.txt"
+
+	flag.Parse()
+
+	var pathArr []string
+
+	if *numberLine && *help {
+		pathArr = os.Args[3:]
+	} else if !*numberLine && !*help {
+		pathArr = os.Args[1:]
+	} else {
+		pathArr = os.Args[2:]
+	}
```

Comment:
There's a handy function `flag.Args()` which returns all of the remaining arguments _after_ any flags that were parsed - this may help you simplify your code here a bit :)